platform-controller/src/main/java/org/hobbit/controller/orchestration/ClusterManager.java

```diff
@@ -0,0 +1,20 @@
+package org.hobbit.controller.orchestration;
+
+import org.hobbit.controller.orchestration.objects.ClusterInfo;
+
+public interface ClusterManager {
```

Comment:
If you have moved the interface to this new package, why did it lost all its Javadoc comments on the way? Please add them back.